### PR TITLE
add support for loadbalancers HTTPS listeners additional certificates

### PR DESCRIPTION
**Pull request by vincentmrg**

### CODE REVIEW

File: loadbalancer_private.tf [hunk 1]
```diff
@@ -159,6 +159,15 @@ resource "aws_lb_listener" "quortex_private_tls" {
   }
 }
 
+# Provides Load Balancer Listener Certificate resources.
+# These resources are for additional certificates and does not replace the default certificate on the listener.
+resource "aws_lb_listener_certificate" "quortex_private" {
+  for_each = var.load_balancer_private_expose_https ? (length(var.load_balancer_private_app_backend_ports) > 0 ? var.load_balancer_private_additional_certs_arns : []) : []
```
Comment (antonincms) [inline loadbalancer_private.tf:165]: I'm not sure to understand what you are doing, are there two ternary here ?
Couldn't you write it like this (or did I misunderstood what you were doing) ?
```
  for_each = var.load_balancer_private_expose_https && length(var.load_balancer_private_app_backend_ports) > 0 ? var.load_balancer_private_additional_certs_arns : []
```

Comment (vincentmrg) [reply loadbalancer_private.tf]: Yes, it's more readable like that... fixed

Comment (antonincms) [reply loadbalancer_private.tf]: I still does not understand, why do you need load_balancer_private_app_backend_ports to set additional certificates ?

Comment (vincentmrg) [reply loadbalancer_private.tf]: It's the actual condition for alb HTTPS listeners creation.

**Review by antonincms — APPROVED** [review]

**Pull request merged**